charts/infra-apps/Chart.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
```

Comment:
The changes section doesn't seem to contain much descriptions of the actual changes. Can you add some m,ore points that summarize the changes to kps and the new versions that are being deployed... like, is there an update to the udnerlying opreator that has it's own release notes and is relevant?